src/stores/Binding/Store.ts

```diff
@@ -377,7 +377,9 @@ const getInitialState = (
 
                 setResourceSchema(
                     data.resource_spec_schema as unknown as Schema
-                );
+                ).catch(() => {
```

Comment:
I _think_ this can just be awaited. The function itself already sets the hydration errors when there is an issue.

Reply:
Okay, perfect. I _thought_ that should be the case. Will replace.